book/tutorials/data-access-and-format/earthaccess.ipynb

```diff
@@ -1749,7 +1749,7 @@
     "- Repository\n",
     "- Access date\n",
     "\n",
-    "Not all of these fields may be available.  But assuming they are available we can automate creating a citation.  I'm going to use the BibTex format. \n",
+    "Not all of these fields may be available.  But assuming they are available we can automate create a citation.  I'm going to use the BibTex format. \n",
```

Comment:
Oooh, tricky.  Maybe "automatically create a citation".  I was trying to avoid "automate the creation of a citation".

Reply:
I like that!